content/go/concepts/composition/composition.md

```diff
@@ -133,6 +133,129 @@ Margherita pizza is a medium pizza with toppings of: [tomatoes mozzarella basil]
 Pizzeria del Corso has a rating of 4 and serves the following pizzas: [{Margherita medium [tomatoes mozzarella basil] true}]
 ```
 
+## Inheritance through Composition
+
+A `struct` can 'inherit' the fields and methods of an embedded `struct` through composition rather than traditional [inheritance](https://www.codecademy.com/resources/docs/general/programming-paradigms/inheritance).
```

Comment:
```suggestion
A struct can 'inherit' the fields and methods of an embedded struct through composition rather than traditional [inheritance](https://www.codecademy.com/resources/docs/general/programming-paradigms/inheritance).
```

Reply:
The struct referenced here doesn't need to be inside backticks. This formatting is used to highlight something referenced in the code examples, such as a variable. Or to offset specific syntax, such as declaring a struct requires the `struct` keyword. This formatting is incorrectly used several times in the entry, please revert the other examples as well.